envs/example/ci-full/group_vars/all.yml

```diff
@@ -16,13 +29,46 @@ neutron:
   lbaas:
     enabled: True
 
+common:
```

Comment:
Work with CI team to get rid of ci-full. We must not be maintaining 2 envs for same purpose.